torch/testing/_internal/common_methods_invocations.py

```diff
@@ -19070,10 +19094,8 @@ def __init__(
         supports_out=True,
         supports_nvfuser=False,
         skips=(
-            # RunTimError: no _refs support for torch.Tensor.masked_select
+            # RunTimeError: no _refs support for torch.Tensor.index_select
```

Comment:
We can live with this one for now. But could you post in the primtorch channel how should this one be implemented (no need to implement it in this PR of course).

Reply:
Will do. FWIW, this also requires matmul which does also not have a ref implementation yet.